app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+
+    for person in people:
+        pers = Person(person["name"], person["age"])
+        result.append(pers)
```

Comment:
Consider using list comprehension to create the list of `Person` instances. This will make the code more concise and efficient. For example: `result = [Person(person['name'], person['age']) for person in people]`.